staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -167,6 +173,8 @@ func (cfg *Config) Complete() CompletedConfig {
 	// the kube aggregator wires its own discovery mechanism
 	// TODO eventually collapse this by extracting all of the discovery out
 	c.GenericConfig.EnableDiscovery = false
+	c.GenericConfig.EnableAggregatedDiscoveryEndpoint = false
```

Comment:
can I get comment about why?  if this is to prevent a conflict with the kube-apiserver's discovery/v1, I would expect it to be set in the `createAggregatorConfig` method in pkg/cmd/kube-apiserver